src/Generator.php

```diff
@@ -246,8 +244,14 @@ public function generateTemplate(string $targetPath, string $templateName, array
         );
     }
 
+    /**
+     * @deprecated MakerBundle only supports AbstractController::class. This method will be removed in the future.
+     * @legacy Remove method in the next major MakerBundle release.
```

Comment:
I think we can just use the first line. I know WE use `@lgacy` to track which things can be removed, but they kind of serve 2 different purposes: (A) `@deprecated` means "Hey user stop using this" while (B) `@legacy` means "Hey future maintainers, don't forget to delete this code someday.

When something is `@deprecated`, let's only put that. This means that, when pruning old features, we just need to look for both tags.

Reply:
good point - done